consoleapp.py

```diff
@@ -0,0 +1,27 @@
+from alonerpg.tablesreader import *
+
+def print_data(data):
+    if isinstance(data, str):
```

Comment:
This could actually be a good opportunity to get rid of the ifs, but maybe we can leave it out od the scope of this PR

Reply:
Next PR then. I'm not sure how to remove the else